app/Http/Requests/Api/Auth/PostLoginRequest.php

```diff
@@ -38,23 +36,25 @@ public function rules(): array
     }
 
     /**
-     * Tap into the parameters to make sure the email is lowercased.
+     * Get all of the input and files for the request.
      *
-     * @param null|array|mixed $keys
+     * @param null|array<int, null|int|string>|mixed $keys
      *
-     * @return array<string, string>
+     * @return array<int|string,mixed>
      */
-    public function all($keys = null): array
+    public function all($keys = null)
     {
-        if ($this->get('email')) {
+        if ($email = $this->get('email')) {
+            $email = is_string($email) ? $email : '';
```

Comment:
`strval` and `larastan` are no longer friends.

<img width="848" alt="Screenshot 2023-05-12 at 19 16 11" src="https://github.com/carbonaut/boilerplate-api-laravel/assets/18501540/4192bc6a-e14a-4f87-8147-5cf9a19d4e4a">